mod/admin.php

```diff
@@ -282,14 +282,14 @@ function admin_page_federation(&$a) {
 	foreach ($platforms as $p) {
 		// get a total count for the platform, the name and version of the
 		// highest version and the protocol tpe
-		$c = q('SELECT COUNT(*) AS `total`, `platform`, `network`, `version` FROM `gserver`
+		$c = qu('SELECT COUNT(*) AS `total`, `platform`, `network`, `version` FROM `gserver`
 				WHERE `platform` LIKE "%s" AND `last_contact` > `last_failure` AND `version` != ""
 				ORDER BY `version` ASC;', $p);
 		$total = $total + $c[0]['total'];
 
 		// what versions for that platform do we know at all?
 		// again only the active nodes
-		$v = q('SELECT COUNT(*) AS `total`, `version` FROM `gserver`
+		$v = qu('SELECT COUNT(*) AS `total`, `version` FROM `gserver`
 				WHERE `last_contact` > `last_failure` AND `platform` LIKE "%s"  AND `version` != ""
```

Comment:
[standards] Please fix SQL query indentation (two tabs instead of one expected) 😊